### PR TITLE
Added student info

### PR DESCRIPTION
**Pull request by vsvipul555**

### CODE REVIEW

File: _data/students.yml [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # New members please add your credentials to a random position in the file.
 # This is to remove conflicts with other people adding their credentials.
 
+- name: Vipul Sharma
+  github: vsvipul555
+  image: vipul.jpg
+  twitter: @VSVipul
```
Comment (shubham-padia) [inline _data/students.yml]: remove `@`

Comment (shubham-padia) [inline _data/students.yml]: Rest all looks good :+1:

Comment (vsvipul555) [inline _data/students.yml]: removed @.. what else do i need to do ?

**Comment by gauthamzz** [comment]
Please do the social media post and finish your registration :+1:  Welcome to OpenCode

**Comment by vsvipul555** [comment]
ok sure

**Pull request merged**